commit 8ddd039e683ef6a1dceab19562c3c34c976b111e (from dc2cb8454c836d083a120ba6f029cae6ca8d1f1f)
Merge: dc2cb84 1e1edea
Author: Roxy-3D <Roxy-3D@users.noreply.github.com>
Date:   Tue Oct 18 15:33:06 2016 -0500

    Merge pull request #5037 from esenapaj/Fix-for-the-commit-dc2cb84
    
    Fix for the commit dc2cb84 (Revert acceleration limiting to avoid overflow)

diff --git a/Marlin/planner.cpp b/Marlin/planner.cpp
index dfde3cc..ffacf76 100644
--- a/Marlin/planner.cpp
+++ b/Marlin/planner.cpp
@@ -1021,8 +1021,8 @@ void Planner::_buffer_line(const float &a, const float &b, const float &c, const
   }
   else {
     #define LIMIT_ACCEL(AXIS) do{ \
-      if (max_acceleration_steps_per_s2[AXIS] < (accel * block->steps[AXIS]) / block->step_event_count)
-        accel = (max_acceleration_steps_per_s2[AXIS] * block->step_event_count) / block->steps[AXIS];
+      if (max_acceleration_steps_per_s2[AXIS] < (accel * block->steps[AXIS]) / block->step_event_count) \
+        accel = (max_acceleration_steps_per_s2[AXIS] * block->step_event_count) / block->steps[AXIS]; \
     }while(0)
 
     // Start with print or travel acceleration

